controllers/lvmcluster_controller.go

```diff
@@ -309,12 +322,15 @@ func (r *LVMClusterReconciler) getRunningPodImage(ctx context.Context) error {
 		// 'POD_NAME' and 'POD_NAMESPACE' are set in env of lvm-operator when running as a container
 		podName := os.Getenv("POD_NAME")
 		if podName == "" {
-			return fmt.Errorf("failed to get pod name env variable")
+			err := fmt.Errorf("failed to get pod name env variable")
+			r.Log.Error(err, "POD_NAME env variable is not set")
+			return err
 		}
 
 		pod := &corev1.Pod{}
 		if err := r.Get(ctx, types.NamespacedName{Name: podName, Namespace: r.Namespace}, pod); err != nil {
-			return fmt.Errorf("failed to get pod %s in namespace %s", podName, r.Namespace)
+			r.Log.Error(err, "failed to get pod")
```

Comment:
- better log at-least pod name

Reply:
agree

Reply:
err will have it, but still i added in the logs again.